README.md

```diff
@@ -123,47 +123,47 @@ SIG-specific GitHub discussions.
+| Java: SDK + Automatic Instrumentation | Every Thursday at 09:00 PT                                                                                 | [Google Doc](https://docs.google.com/document/d/1WK9h4p55p8ZjPkxO75-ApI9m0wfea6ENZmMoFRvXSCw/)                            | [#otel-java](https://cloud-native.slack.com/archives/C014L2KCTE3)                                                                                            |
+| JavaScript: SDK                       | Every Wednesday at 09:00 PT                                                                                | [Google Doc](https://docs.google.com/document/d/1tCyoQK49WVcE-x8oryZOTTToFm7sIeUhxFPm9g-qL1k)                             | [#otel-js](https://cloud-native.slack.com/archives/C01NL1GRPQR) and [GitHub Discussions](https://github.com/open-telemetry/opentelemetry-js/discussions)     |
+| PHP: SDK                              | Every Wednesday at 10:30 PT                                                                                | [Google Doc](https://docs.google.com/document/d/1WLDZGLY24rk5fRudjdQAcx_u81ZQWCF3zxiNT-sz7DI/edit)                        | [#otel-php](https://cloud-native.slack.com/archives/C01NFPCV44V)                                                                                             |
+| Python: SDK                           | Every Thursday at 09:00 PT                                                                                 | [Google Doc](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit)                        | [#otel-python](https://cloud-native.slack.com/archives/C01PD4HUVBL)                                                                                          |
+| Ruby: SDK                             | Every Tuesday at 09:00 PT                                                                                  | [Google Doc](https://docs.google.com/document/d/1D15bO8o340sQm2CVZiukEJuCO_XMMHKPuTznoEhyFqE/edit)                        | [#otel-ruby](https://cloud-native.slack.com/archives/C01NWKKMKMY) and [GitHub Discussions](https://github.com/open-telemetry/opentelemetry-ruby/discussions) |
+| Rust: SDK                             | Every Tuesday at 16:00 UTC                                                                                 | [Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)                             | [#otel-rust](https://cloud-native.slack.com/archives/C03GDP0H023) and [Gitter](https://gitter.im/open-telemetry/opentelemetry-rust)                          |
```

Comment:
@open-telemetry/rust-maintainers is Gitter still in use? The last message there was sent 8 months ago. You could consider archiving it in favor of Slack and removing the link here in a follow-up PR.